server/boot/user.js

```diff
@@ -608,7 +608,14 @@ module.exports = function(app) {
     return User.findById(req.accessToken.userId, function(err, user) {
       if (err) { return next(err); }
       return user.updateAttribute('password', password, function(err) {
-        if (err) { return next(err); }
+        if (err) {
```

Comment:
The intention was to pass the message created in `user.js` with `validatesUniquenessOf` API:
```
message: 'Duplicate email exists, please contact team@freeCodeCamp.com'
```